### PR TITLE
!!!TASK: Upgrade to latest neos/behat

### PR DESCRIPTION
**Pull request by johannessteu**
Distribution for travis https://travis-ci.com/johannessteu/neos-development-distribution/builds

### CODE REVIEW

**Comment by johannessteu** [comment]
Depends on https://github.com/neos/behat/pull/14

File: Neos.Flow/Tests/Behavior/Features/Bootstrap/SubProcess/SubProcess.php [hunk 1]
```diff
@@ -81,10 +81,10 @@ public function quit()
     /**
      * Launch sub process
      *
-     * @return array The new sub process and its STDIN, STDOUT, STDERR pipes – or false if an error occurred.
+     * @return array|bool The new sub process and its STDIN, STDOUT, STDERR pipes – or false if an error occurred.
      * @throws \RuntimeException
      */
-    protected function launchSubProcess()
+    protected function launchSubProcess(): ?array
```
Comment (kdambekalns) [inline Neos.Flow/Tests/Behavior/Features/Bootstrap/SubProcess/SubProcess.php]: Does that return type declaration work if a bool is returned? Seems weird to me.

Comment (johannessteu) [reply Neos.Flow/Tests/Behavior/Features/Bootstrap/SubProcess/SubProcess.php]: Nope. Overlooked the false option in return

**Comment by johannessteu** [comment]
All related PR's:
https://github.com/neos/behat/pull/14
https://github.com/neos/neos-development-collection/pull/2343
https://github.com/neos/flow-development-collection/pull/1496

File: Neos.Flow/Tests/Behavior/Features/Bootstrap/SecurityOperationsTrait.php [hunk 2]
```diff
@@ -106,7 +111,6 @@ public static function cleanUpSecurity()
 
     /**
      * @Given /^I am not authenticated$/
-     * @throws \Neos\Flow\ObjectManagement\Exception\UnknownObjectException
```
Comment (bwaidelich) [inline Neos.Flow/Tests/Behavior/Features/Bootstrap/SecurityOperationsTrait.php]: Why did you remove all those `@throws`  annotations?

Comment (johannessteu) [reply Neos.Flow/Tests/Behavior/Features/Bootstrap/SecurityOperationsTrait.php]: Probably ide auto stuff since they are not thrown in that method but somwhere deeper?
